task/ecosystem-cert-preflight-checks/OWNERS

```diff
@@ -1,2 +1 @@
-
-opdev
+#opdev
```

Comment:
I think the OWNERS should be @skattoju (anyone else?)

Reply:
Does it have to be github usernames ? We have `@preflight-dev` in slack that pings preflight team members. I had initial set it as opdev to reflect this github org https://github.com/opdev Other team members include @acornett21 @bcrochet @komish